wca-regulations.md

```diff
@@ -104,7 +104,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 2l) A competitor may be disqualified immediately, or after a warning, depending on the nature and severity of the infraction.
     - 2l1) A disqualified competitor is not eligible for the refund of any expenses due to participating in the competition.
 - 2n) Competitors may verbally dispute a ruling to the WCA Delegate.
-    - 2n1) Disputes are only permitted during the competition, within 30 minutes after the disputed incident happened and before the start of any following rounds of the relevant event.
+    - 2n1) Disputes are only permitted during the competition (see [Regulation 9u](regulations:regulation:9u)), within 30 minutes after the disputed incident happened and before the start of any following rounds of the relevant event.
```

Comment:
@lgarron Is the intention for this regulation to have the extra (serial) comma there? 
```suggestion
    - 2n1) Disputes are only permitted during the competition (see [Regulation 9u](regulations:regulation:9u)), within 30 minutes after the disputed incident happened, and before the start of any following rounds of the relevant event.
```

Reply:
Yeah, and in fact this should be "or" (not "and").

```suggestion
    - 2n1) Disputes are only permitted during the competition (see [Regulation 9u](regulations:regulation:9u)), within 30 minutes after the disputed incident happened, or before the start of any following rounds of the relevant event.
```